src/pane-container.coffee

```diff
@@ -164,6 +164,17 @@ class PaneContainer extends Model
     else
       false
 
+  moveActiveItemToPane: (destPane) ->
+    item = @activePane.getActiveItem()
+    @activePane.moveItemToPane(item, destPane)
+    destPane.setActiveItem(item)
+    destPane.focus()
```

Comment:
I definitely agree. The only reason that led me to make this change is that it felt _better_ when writing specs. I can remove/update the assertions on the active pane, though.
I'll change it back right away.